Packs/Legacy/Integrations/Lastline_v2/Lastline_v2.py

```diff
@@ -159,7 +159,10 @@ def report_generator(result: Dict, threshold=None):
     score = result['data'].get('score')
     uuid = result['data'].get('task_uuid')
     submission_time = result['data'].get('submission')
-    indicator = context_entry.get('DBotScore', {}).get('Indicator', 'None')
+    if key == 'File':
+        indicator = context_entry.get('DBotScore', [{}])[0].get('Indicator', 'None')
+    else:
+        indicator = context_entry.get('DBotScore', {}).get('Indicator', 'None')
```

Comment:
why?

Reply:
cause the function get_report_context return a list if the type is File